tests/auth_tests/test_hashers.py

```diff
@@ -433,6 +433,13 @@ def test_check_password_calls_harden_runtime(self):
             check_password('wrong_password', encoded)
             self.assertEqual(hasher.harden_runtime.call_count, 1)
 
+    def test_non_string_or_bytes_pass(self):
+        with self.assertRaisesMessage(
```

Comment:
Use `msg` variable and this style: https://github.com/django/django/blob/55cdf6c52db07f29128741b8734a523ed042e465/tests/many_to_one/tests.py#L53-L55